spec/services/post_alerter_spec.rb

```diff
@@ -348,7 +348,7 @@ def create_post_with_alerts(args = {})
     fab!(:topic) { Fabricate(:topic, category: category) }
 
     it 'does not notify for muted users' do
-      post = Fabricate(:post, raw: '[quote="EvilTrout, post:1"]whatup[/quote]', topic: topic)
+      post = Fabricate(:post, raw: '[quote="eviltrout, post:1"]whatup[/quote]', topic: topic)
```

Comment:
Is it necessary to modify the existing specs? If the PR only fixes notifications when prioritizing full names, then existing specs shouldn't be changed, right?